packages/opentelemetry-cloud-monitoring-exporter/test/monitoring.test.ts

```diff
@@ -177,48 +171,42 @@ describe('MetricExporter', () => {
         unhandledPromiseRejectionEvent = true;
       });
 
-      await exporter.export(records, () => {});
+      await new Promise<ExportResult>(resolve => {
```

Comment:
Should we use util.promisify instead?

Reply:
I actually tried that, but it won't work correctly because the signature used in the upstream SDK doesn't follow the [error first convention](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original:~:text=Takes%20a%20function%20following%20the%20common%20error%2Dfirst%20callback%20style%2C%20i.e.%20taking%20a%20(err%2C%20value)%20%3D%3E%20...%20callback%20as%20the%20last%20argument%2C%20and%20returns%20a%20version%20that%20returns%20promises.):

```ts
function export(
  metrics: ResourceMetrics,
  resultCallback: (result: {code: ExportResultCode, error?: Error}) => void
): void;
```

since there is only one argument `result` to the callback, it is interpreted as the error. IOW, typescript selects [this overload](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/adc1b6dd3233c3478e0744c0112355b8e2b431a1/types/node/util.d.ts#L1009):

```ts
export function promisify(fn: (callback: (err?: any) => void) => void): () => Promise<void>;
```